style.css

```diff
@@ -180,16 +180,16 @@ h1 {
   float: right;
   font-weight: 500;
   font-size: 16px;
-  background-color: black;
+  background-color: var(--onyx-black);
   border: none;
-  color: #FF00FF;
+  color: var(--sharp-retroPink);
   cursor: pointer;
   outline: none;
   width: 15%;
   height: 36px;
   border-radius: 0px;
   border-style: solid;
-  border-color: #FF00FF;
+  border-color: var(--sharp-retroPink);
```

Comment:
Thanks for switching to the route codes (I noticed a few of mine were still using hex codes).

Reply:
your very welcome 😸